tests/python_client/milvus_client/test_milvus_client_insert.py

```diff
@@ -165,12 +163,12 @@ def test_milvus_client_insert_data_invalid_type(self, data):
         # 1. create collection
         client_w.create_collection(client, collection_name, default_dim, consistency_level="Strong")
         # 2. insert
-        error = {ct.err_code: 1, ct.err_msg: f"None rows, please provide valid row data."}
+        error = {ct.err_code: 999, ct.err_msg: f"wrong type of argument 'data',expected 'Dict' or list of 'Dict', got 'str'"}
         client_w.insert(client, collection_name, data,
                         check_task=CheckTasks.err_res, check_items=error)
 
     @pytest.mark.tags(CaseLabel.L1)
-    @pytest.mark.xfail(reason="pymilvus issue 1895")
+    # @pytest.mark.xfail(reason="pymilvus issue 1895")
```

Comment:
Why was this line of code just commented out

Reply:
it can deleted in next pr